ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -35,16 +35,15 @@ cp "${SHARED_DIR}/install-config.yaml" "${dir}/"
 mkdir -p ~/.ssh
 cp "${SSH_PRIV_KEY_PATH}" ~/.ssh/
 
-
-TF_LOG=debug openshift-install --dir="${dir}" create manifests
+openshift-install --dir="${dir}" create manifests &
```

Comment:
Is this a change that we should make in all of the places where `openshift-install create manifests` is called?

```
$ grep -rnE ./ci-operator/ -e "openshift-install (.* )?create manifests"
./ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml:439:        TF_LOG=debug openshift-install --dir=/tmp/artifacts/installer create manifests --log-level=debug
./ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml:1136:          openshift-install --dir=/tmp/artifacts/installer create manifests
./ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml:1163:          openshift-install --dir=/tmp/artifacts/installer create manifests
./ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml:1221:            openshift-install --dir=/tmp/artifacts/installer create manifests
./ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml:1267:            openshift-install --dir=/tmp/artifacts/installer create manifests
./ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml:770:            openshift-install --dir=/tmp/artifacts/installer/ create manifests
./ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml:779:          openshift-install --dir=/tmp/artifacts/installer/ create manifests
./ci-operator/templates/openshift/installer/cluster-launch-installer-src.yaml:555:            openshift-install --dir=/tmp/artifacts/installer/ create manifests
./ci-operator/templates/openshift/installer/cluster-launch-installer-metal-e2e.yaml:402:        openshift-install --dir=/tmp/artifacts/installer/ create manifests
./ci-operator/templates/openshift/installer/cluster-launch-installer-custom-test-image.yaml:644:            openshift-install --dir=/tmp/artifacts/installer/ create manifests
./ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-e2e.yaml:385:        TF_LOG=debug openshift-install --dir=/tmp/artifacts/installer create manifests --log-level=debug
./ci-operator/templates/openshift/installer/cluster-launch-installer-upi-src.yaml:417:          openshift-install --dir=/tmp/artifacts/installer create manifests
./ci-operator/templates/openshift/installer/cluster-launch-installer-upi-src.yaml:442:          openshift-install --dir=/tmp/artifacts/installer create manifests
./ci-operator/templates/openshift/installer/cluster-launch-installer-upi-src.yaml:501:            openshift-install --dir=/tmp/artifacts/installer create manifests
./ci-operator/templates/openshift/installer/cluster-launch-installer-remote-libvirt-e2e.yaml:518:        mock-nss.sh openshift-install create manifests --dir=/tmp/artifacts/installer
./ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh:39:TF_LOG=debug openshift-install --dir="${dir}" create manifests
```